external-crates/move/crates/move-analyzer/tests/colon_colon_completion.snap

```diff
@@ -1322,6 +1369,19 @@ EnumMember 'SomePositionalVariant(..)'
     INSERT TEXT: 'SomePositionalVariant(${1}, ${2})'
 EnumMember 'SomeVariant'
 
+-- test 38 -------------------
+use line: 96, use_col: 27
+Module 'Completion'
+Module 'colon_colon'
+Module 'dot'
+Module 'init'
+Module 'init_otw'
+Module 'macro_dot'
+Module 'object'
+Module 'other_mod_dot'
+Module 'uses'
+Struct 'SomeStructComp'
```

Comment:
This is the important part - module member on the list of suggestions in addition to modules themselves